babelsberg/constraintinterpreter.js

```diff
@@ -258,63 +257,31 @@ Object.subclass('Babelsberg', {
         func.onError = opts.onError;
 
         solvers = this.filterSolvers(solvers, opts);
-
-        solvers.each(function(solver) {
+        var constraints = this.createEquivalentConstraints(solvers, opts, func, errors);
+        var constraint = this.chooseConstraint(constraints, opts, errors);
+        if (!opts.postponeEnabling && constraint) {
             try {
-                var constraint = solver.always(Object.clone(opts), func);
-                constraint.logTimings = opts.logTimings;
-                constraints.push(constraint);
-            } catch (e) {
-                errors.push(e);
-                return false;
-            }
-        });
-
-        if (constraints.length > 1) {
-            for (var i = 0; i < constraints.length; i++) {
+                constraint.isAnyVariableCurrentlySuggested = true; // do not increase
```

Comment:
this name... it seems wrong. ``suggestValue`` does not set this to true, instead ``always`` does?